avocado/core/remote/runner.py

```diff
@@ -64,6 +64,8 @@ def _copy_files(self):
         paths = set()
         for i in xrange(len(self.job.urls)):
             url = self.job.urls[i]
+            if ':' in url:
```

Comment:
Because of a typo in the commit message, I spent some time trying to find where/how this patch added a `_copy_files()` call/check/etc.  It should read: "This patch adds a check to _copy_files()...`

Then about the patch itself, it LGTM.  Let me do some functional testing.